Algorithm.Python/NamedArgumentsRegression.py

```diff
@@ -0,0 +1,59 @@
+﻿# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Indicators")
+AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Data import *
+from QuantConnect.Algorithm import *
+from QuantConnect.Indicators import *
+import numpy as np
+
+### <summary>
+### Demonstration of requesting daily resolution data for US Equities.
+### This is a simple regression test algorithm using a skeleton algorithm and requesting daily data.
+### </summary>
+### <meta name="tag" content="using data" />
+class NamedArgumentsRegression(QCAlgorithm):
+    '''Regression algorithm that makes use of PythonNet kwargs'''
+
+    def Initialize(self):
+        '''Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.'''
+
+        self.SetStartDate(2013,10,8)   #Set Start Date
+        self.SetEndDate(2013,10,17)    #Set End Date
+        self.SetCash(100000)           #Set Strategy Cash
+
+        # Use named args for addition of this security to our algorithm
+        symbol = self.AddEquity(ticker="SPY", resolution=Resolution.Daily).Symbol
```

Comment:
Let's change the order and assert it still works & would be nice to make **all** calls use named arguments really 🚀